endpoint ownership --> endpoint validity

scharan opened the pull request:
This mechanism does not really prove ownership. It just ensures that the endpoint passes the AEG handshake. This is simply a technical nit-pick.

azurecla commented:
Hi __@v-micada__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-micada). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

scharan committed:
endpoint ownership --> endpoint validity

This mechanism does not really prove ownership. It just ensures that the endpoint passes the AEG handshake. This is simply a technical nit-pick.

banisadr reviewed (changes requested):
I understand your point (I assume) that ownership may not be the best term, however validity is not ideal either. Making a simple POST to many websites such as bing.com will result in a 200 response as it is a valid endpoint, but as you do not have ownership of the endpoint, we do not want to allow you to subscribe bing.com to an event subscription.
